packages/ui/src/components/Navbar/ProfileInfo/ProfileInfo.tsx

```diff
@@ -8,37 +8,40 @@ const ProfileInfo: React.FC<{
   name: string | null;
   email: string;
   imageUrl: string | null;
-}> = ({ id, name, email, imageUrl }) => {
+  isLargeScreen?: boolean;
```

Comment:
This component will not be used anywhere else other than the web platform. You can use media query directly in the component.